src/R/Editor/Impl/Completion/Providers/PackageFunctionCompletionProvider.cs

```diff
@@ -60,8 +60,12 @@ public class FunctionCompletionProvider : IRCompletionListProvider {
                 IEnumerable<INamedItemInfo> functions = pkg.Functions;
                 if (functions != null) {
                     foreach (INamedItemInfo function in functions) {
-                        bool? isSnippet = infoSource?.IsSnippet(function.Name);
-                        if (!isSnippet.HasValue || !isSnippet.Value) {
+                        bool isSnippet = false;
+                        // Snippets are suppressed if user typed namespace
+                        if (!context.IsInNameSpace() && infoSource != null) {
```

Comment:
This is [mostly] cosmetic however. By default snippet wins so when typing lm icon with scissors is visible. However, snippet is only inserted if user hits tab-tab. Otherwise there is no difference and typing lm( brings regular function. When in namespace snippets are suppressed and icon looks like function. So yes, code should suppress snippets past selectors as well although there is not much harm at the moment, perhaps occasional wrong icon.